packages/salesforcedx-vscode-core/test/integration/dummy.test.ts

```diff
@@ -0,0 +1,2 @@
+describe('No tests yet', () => {
```

Comment:
Is this one necessary?

Reply:
Mocha complained otherwise:
```
Warning: Could not find any test files matching pattern: out/test/integration
No test files found

=============================== Coverage summary ===============================
Statements   : Unknown% ( 0/0 )
Branches     : Unknown% ( 0/0 )
Functions    : Unknown% ( 0/0 )
Lines        : Unknown% ( 0/0 )
================================================================================
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! salesforcedx-vscode-core@49.10.0 test:integration: `node ./node_modules/cross-env/dist/bin/cross-env.js VSCODE_NLS_CONFIG={} ./node_modules/nyc/bin/nyc.js ./node_modules/mocha/bin/_mocha --recursive out/test/integration --reporter mocha-multi-reporters --reporter-options configFile=../../config/mochaIntegrationTestsConfig.json`
npm ERR! Exit status 1```

but there might be a cleaner option to avoid it.

Reply:
If there's no tests after you move that test, could we remove `test:integration` from the npm script?

Reply:
Your call :-)

Should all tests be vscode tests anyway?  If "yes", then I'd remove it.
Would leaving the dummy test help devs add non-vscode tests?  If "yes", I'd leave it.

Reply:
I feel that we should remove that, and add back if we ever find a need :) imo we won't, and if we are going to test anything non-specific to vscode let's resort to unit tests

Reply:
👍  done